### DISCUSSION TITLE
Obsidian Attachment Management

### DISCUSSION THREAD

**Discussion by Hela-Girl** [trganda/obsidian-attachment-management#79] [General]
Everything would be fine, but it does not rename folders, only files ((
https://i.imgur.com/xrcr5dZ.jpg
https://i.imgur.com/Jlzw8WC.jpg

Can I ask you for the ability to add dates to folders?

**Comment by trganda** [comment]
`${data}` was not supported in the folder, I will add this feature in the next version.

**Comment by Hela-Girl** [comment]
It would be great if you could add it. I really need this function for folders.
May I know when are you planning a new version?

**Reply by trganda** [reply]
You will see that this weekend, please be patient 😄

**Reply by Hela-Girl** [reply]
Thanks!

**Reply by trganda** [reply]
Hi @Hela-Girl , you can use ${data} in folder setting by upgrade to latest version

**Reply by Hela-Girl** [reply]
Thank you very much, it works great!

**Reply by Hela-Girl** [reply]
https://i.imgur.com/pUhw9KY.jpg

It’s a pity there’s not enough file counter, but I solved it using IMG-${md5}. Thank you again!

**Comment by teddychn** [comment]
Renaming a note will rename relevant folder name. Great! 

But I find it create a new folder, moving attached file to the new folder, and leave the old-blank folder there. Could it delete the old-blank folder as well? Thanks!
